lib/translator/src/Controller.php

```diff
@@ -51,6 +51,7 @@ class Controller extends PhpObj {
         '\mod_quiz\event\attempt_preview_started' => 'AttemptStarted',
         '\mod_quiz\event\attempt_reviewed' => ['AttemptReviewed', 'QuestionSubmitted'],
         '\mod_quiz\event\attempt_viewed' => 'ModuleViewed',
+        '\mod_quiz\event\attempt_submitted' => ['AttemptSubmitted', 'QuestionSubmitted'],
```

Comment:
Should Question Submitted be here?

Reply:
I'm not sure. I made it as close to the attempt_reviewed as possible. I thought it was what was sending the statement about the individual questions.